tests/cli/commands/remote_commands/test_config_command.py

```diff
@@ -447,3 +445,30 @@ def test_lint_detects_configs_with_env_vars(self, env_var, config_change, expect
 
         assert expected_message in normalized_output
         assert config_change.suggestion in normalized_output
+
+    def test_lint_detects_default_value_change_upgrade(self):
```

Comment:
Also means this test should be refactored and moved into [test_configuration.py](https://github.com/apache/airflow/blob/main/tests/core/test_configuration.py).